src/Mono.Android/Android.Runtime/JavaProxyThrowable.cs

```diff
@@ -38,6 +39,10 @@ public static JavaProxyThrowable Create (Exception innerException)
 
 		void TranslateStackTrace ()
 		{
+			[UnconditionalSuppressMessage ("Trimming", "IL2026", Justification = "StackFrame.GetMethod() is \"best attempt\", we handle null & exceptions")]
+			static MethodBase? StackFrameGetMethod (StackFrame frame) =>
```

Comment:
Will this reason hold for actual usage?
For example - in OTel, we had a similar issue, and the outcome of the discussion was that it's very important to get something for each frame. So there we rely on .ToString() on the `StackFrame` which works even in NativeAOT/trimming (unless one explicitly disable stack frame info in NativeAOT as a size optimization).

I'm not saying this is wrong, just asking where is this used, and if we would like to get at least some info all the time in those scenarios.

Reply:
Let me investigate if we could just use `StackFrame.ToString()` for this. It's possible we would get the same text, and the tests would pass: https://github.com/xamarin/xamarin-android/commit/1aa0ea7a8aef616d00e8467aa9eec83a1571cbd0

Reply:
I tested in a console app, and it appears that `StackFrame.GetMethod()` returns `null` under NativeAOT. But calling `StackFrame.ToString()` returns useful information that contains the method name, example:
```
MainActivity.OnCreate() + 0x37 at offset 55 in file:line:column <filename unknown>:0:0
```

To support that one day, we _could_ parse the string, but I think delaying that work for now is best.

I left a code comment for now, linking to: 
* https://github.com/xamarin/xamarin-android/issues/8724